ivy/functional/backends/paddle/statistical.py

```diff
@@ -124,12 +124,12 @@ def cumsum(
     dtype: Optional[paddle.dtype] = None,
     out: Optional[paddle.Tensor] = None,
 ) -> paddle.Tensor:
-    raise IvyNotImplementedException()
+    return paddle.cumsum(x, axis=axis, dtype=dtype)
 
 
 def einsum(
     equation: str,
     *operands: paddle.Tensor,
     out: Optional[paddle.Tensor] = None,
 ) -> paddle.Tensor:
-    raise IvyNotImplementedException()
+    return paddle.einsum(equation, *operands)
```

Comment:
Please verify the results with another backend to make sure the implementation is correct